__tests__/app.test.js

```diff
@@ -21,7 +22,7 @@ describe("games api errors", () => {
   });
 });
 
-describe("GET /api/categories", () => {
+describe("3.GET /api/categories", () => {
```

Comment:
```js
describe("3.GET /api/categories", () => {
```
Do the numbers that you've added represent the ticket numbers? In general ticket numbers won't be sequential and you will be adding new tests to existing code bases so they won't always line up nicely. They look nice in this one because we've set them out to be in order but the tests should be independent of them and just describe the code they're testing.